driver/normalizer/parser.go

```diff
@@ -63,17 +63,18 @@ var ToNoder = &native.ObjectToNoder{
 		"Compare":  {"comparators": true, "ops": true},
 		// FIXME: check call.keywords
 		//"Call"                    : { "args": true, "keywords": true},
-		"With":                    {"body": true, "items": true},
-		"FunctionDef":             {"body": true, "decorator_list": true},
-		"Lambda":                  {"body": true},
-		"arguments":               {"defaults": true},
-		"Try":                     {"body": true, "orelse": true, "finalbody": true, "handlers": true},
```

Comment:
It means that `Try.handlers` are not promoted to node. https://github.com/bblfsh/python-driver/pull/80/files#diff-56de3fb994af9b7ff9f631ea2d5aec42 shows how the uast changes in this case.